docs/advanced-topics/metrics.rst

```diff
@@ -1,99 +1,147 @@
 
-Why would I want to submit workflow metrics?
---------------------------------------------
-As a platform owner, workflow metrics indicate to others that your platform is compatible with many workflows on Dockstore. Workflow metrics provide valuable information to users, including information about the resources and time needed to run the workflow. It helps the user determine if the workflow is high quality and is very likely to work for others.
-
 .. _How to view metrics:
 
 How do I view workflow metrics?
 -------------------------------
 
-To view workflow metrics for a workflow on Dockstore, you will need to use the ``metrics`` feature flag.
+To search for workflows with metrics, navigate to the `search <https://dockstore.org/search>`_ page and select a platform for the Execution Metrics and/or Validation Metrics facets.
 
-To use the ``metrics`` feature flag, append ``metrics`` to the Dockstore URL as a query parameter. You only need to do this once, unless you refresh/close your browser.
+.. figure:: /assets/images/docs/metrics/metrics-search-facets.png
+    :alt: Metrics search facets
 
-For example, if you're on the https://dockstore.org page, append ``metrics`` such that it looks like this: https://dockstore.org?metrics.
+Select a workflow and click on the Versions tab. Versions that have metrics will have a check mark in the Metrics column.
 
-If you're on a page that already contains query parameters, indicated by the presence of a question mark, append ``metrics`` to the URL using an ampersand. For example, if you're on the https://dockstore.org/workflows/github.com/gatk-workflows/seq-format-conversion/BAM-to-Unmapped-BAM:3.0.0?tab=info page, append ``metrics`` such that it looks like https://dockstore.org/workflows/github.com/gatk-workflows/seq-format-conversion/BAM-to-Unmapped-BAM:3.0.0?tab=info&metrics.
+.. figure:: /assets/images/docs/metrics/versions-metrics-column.png
+    :alt: Metrics column in Versions table
 
-After applying the ``metrics`` feature flag, a Metrics tab will appear when viewing a workflow on Dockstore. Click on the Metrics tab and you will see workflow metrics if they are available.
+Select a version with metrics then click on the Metrics tab to view the metrics available.
 
-.. figure:: /assets/images/docs/submit-metrics/metrics-tab.png
+.. figure:: /assets/images/docs/metrics/metrics-tab.png
     :alt: Metrics tab
 
+Why would I, a platform owner, want to submit workflow metrics?
+---------------------------------------------------------------
+
+As a platform owner, workflow metrics indicate to others that your platform is compatible with many workflows on Dockstore. Workflow metrics provide valuable information to users, including information about the resources and time needed to run the workflow. It helps the user determine if the workflow is high quality and is very likely to work for others.
```

Comment:
Nit: I would say "likely to work" instead of "very likely to work" (although I see it was already like this).